### DISCUSSION TITLE
Account got suspended and support ignores me

### DISCUSSION THREAD

**Discussion by Ciastekbatak** [community/community#122112] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

2 Week ago my account got suspended for "suspicious activities in my account" and then I opened ticket because there is nothing malicious going on and till then support ignores me my account is https://github.com/CiastekbatakPro

**Comment by ebndev** [comment]
Hi @Ciastekbatak,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
